accord-maelstrom/src/test/java/accord/maelstrom/Runner.java

```diff
@@ -185,7 +185,14 @@ void run(String ... commands) throws IOException
         {
             logger.info("Seed {}; rerun with -D{}={}", seed, key, seed);
             RandomSource randomSource = new DefaultRandom(seed);
-            Runner.run(nodeCount, new StandardQueue.Factory(randomSource), randomSource::fork, factory, commands);
+            try
+            {
+                Runner.run(nodeCount, new StandardQueue.Factory(randomSource), randomSource::fork, factory, commands);
```

Comment:
these tests are not deterministic, so same seed doesn't actually get the same results =(